api/nodes/serializers.py

```diff
@@ -1566,15 +1568,12 @@ def get_type(request):
             return get_kebab_snake_case_field(request.version, 'draft-registrations')
 
 
-class DraftRegistrationDetailSerializer(DraftRegistrationSerializer):
+class DraftRegistrationDetailLegacySerializer(DraftRegistrationSerializerLegacy):
     """
-    Overrides DraftRegistrationSerializer to make id required.
+    Overrides DraftRegistrationSerializerLegacy to make id required.
     registration_supplement cannot be changed after draft has been created.
 
     Also makes registration_supplement read-only.
-
-    Either pass in registration_metadata (old workflow) or registration_responses
```

Comment:
Add back - removed in merge w/ schemablocks

Reply:
🆗

Reply:
🆗